ocs_ci/ocs/openshift_lightspeed.py

```
@@ -0,0 +1,352 @@
+           granting it the ``lightspeed-operator-query-access`` ClusterRole, and
+           minting a short-lived token via ``oc create token``.
+
+        Returns:
+            str: Bearer token string.
+        """
+        if not self._token:
+            ocp = OCP(namespace=self.namespace)
+            try:
+                self._token = ocp.get_user_token()
+                logger.info("Retrieved OLS bearer token via oc whoami --show-token")
+            except Exception:
```

Ruff (0.16.3)

[warning] Do not catch blind exception: `Exception`

(BLE001)